data/text_data/fr_FR.yaml

```
@@ -183,6 +6056,80 @@
   pretty: la <Psysalis de la Déesse>
   cryptic:
 
+
+
+- name: Faron Trial Fi Questions Text
+  # 350 L3
+  standard: "Lorsque le <y<réceptacle spirituel>> sera rempli, votre âme\ns'épanouira et la Déesse vous octroiera un <r<nouveau\npouvoir>>.\n\n
+    Avez‐vous des questions, Maître ?\n[1-]Oui[2]Non"
+
+- name: Trial Direct Nothing
+  standard: " La psysalis que ce chant ouvre ne vous apportera <r<rien>>."
+
+- name: Trial Direct One
+  standard: " La psysalis que ce chant ouvre renferme <item>."
+
+- name: Trial Direct Two
+  standard: " La psysalis que ce chant ouvre renferme <item> et <r<1 autre objet>>."
+
+- name: Trial Direct Three Plus
+  standard: " La psysalis que ce chant ouvre renferme <item> et <r<%d autres objets>>."
+
+- name: Trial Useless
+  standard: " Vous n'en aurez proabablement aucun usage..."
+
+- name: Trial Useful
+  standard: " Il semblerait que ce chant vous permette d'accéder à des objets utiles..."
+
+- name: Trial Required
+  standard: " Votre âme s'épanouira à l'entrée de la psysalis que ce chant ouvre."
+
+- name: Farore's Courage Inventory
+  standard: Ce chant ouvre la psysalis située dans la <b+<forêt de Firone>>.
+
+- name: Din's Power Inventory
+  standard: Ce chant ouvre la psysalis située dans le <b+<volcan d'Ordinn>>.
+
+- name: Nayru's Wisdom Inventory
+  standard: Ce chant ouvre la psysalis située dans le <b+<désert de Lanelle>>.
+
+- name: Song of the Hero Inventory
+  standard: Ce chant ouvre la psysalis de la Déesse à <b+<Célesbourg>>.
+
+- name: Impa SoT Text
+  # 502 L7
+  standard: "N'aie aucune crainte. J'assurerai la protection\nde dame <color blue<Zelda>coloroff>.
+    Rends-toi à présent en <regions>. L'<r<objet nécessaire à ta destinée>> \nt'y attend."
+
+- name: First Groosenator Choice Text
+  # 501 L101
+  standard: "Alors, où est‐ce que je te jette ?\n[1]Ciel[2]Surface[3]<b<forêt de Firone\ninondée>>[4-]Nulle part"
+
+- name: Groosenator can't fling text
+  standard: "Désolé, <heroname>. Je ne peux pas te catapulter\n en un lieu où tu n'as jamais été."
+
+- name: FFW Bucha leave question text
+  standard: "<action6 \x03>L'eau n'est-elle pas à ton goût?\nJe connais un passage secret pour sortir de la forêt,\nveux-tu que je te le montre ?\n[1]S'il te plaît[2-]Non merci"
```

```suggestion
- name: FFW Bucha leave question text
  standard: "<action6 \x03>L'eau n'est-elle pas à ton goût?\nJe connais un passage secret pour sortir de la\nforêt, veux-tu que je te le montre ?\n[1]S'il te plaît[2-]Non merci"
```

Needs newline moving.